app/config.go

```diff
@@ -23,13 +24,17 @@ import (
 // - Address: The external address of the conode, used by others to connect to this one
 // - ListenAddress: The address this conode is listening on
 // - Description: The description
+// - WebSocketTLSCertificate: TLS certificate for the WebSocket
+// - WebSocketTLSCertificateKey: TLS certificate key for the WebSocket
 type CothorityConfig struct {
-	Suite         string
-	Public        string
-	Private       string
-	Address       network.Address
-	ListenAddress string
-	Description   string
+	Suite                      string
+	Public                     string
+	Private                    string
+	Address                    network.Address
+	ListenAddress              string
+	Description                string
+	WebSocketTLSCertificate    CertificateURL
```

Comment:
There needs to be some new code in config_test.go proving that YAML can handle string:foo\nbar\nbaz correctly, because PEM-encoded certs have newlines in them.

Reply:
I guess you mean TOML, not YAML. But that's a fair concern. By checking the doc (https://github.com/toml-lang/toml) it seems it should not be an issue, right? I can still write a test for that (`ParseCothority`), since there isn't a test for loading and parsing the `private.toml` file in `config_test.go` anyway.